newlib/libc/stdio/nano-vfprintf_i.c

```diff
@@ -104,6 +105,20 @@ _printf_common (struct _reent *data,
 error:
   return -1;
 }
+
+// Not defined in the ESP8266 Arduino
```

Comment:
I think the reference to Arduino might not be appropriate, as this project is not arduino-specific.

Reply:
Ah, so newlib is also used by the SDK as well?  Just checking the reasoning as I'm only testing w/Arduino SDK now (but can't imagine any meaningful differences).

Reply:
It's not, but i have a few personal projects which do not use Arduino and use this newlib fork, so I'd like to keep it generic, if reasonable.